src/components/manage-registry/manage-identities/ManageIdentities.tsx

```diff
@@ -0,0 +1,213 @@
+import { Button, Flex, Input, Box } from "@chakra-ui/react";
+import { useState, useEffect } from "react";
+import { useMutation } from "@tanstack/react-query";
+import { ethers } from "ethers";
+import { EvmAddress, UpdateIdentityCountryProps } from "@/types/types";
+import { MenuSelect } from "@/components/MenuSelect";
+import {
+  writeIdentityRegistryDeleteIdentity,
+  writeIdentityRegistryUpdateCountry,
+  writeIdentityRegistryUpdateIdentity,
+} from "@/services/contracts/wagmiGenActions";
+import { useWalletInterface } from "@/services/wallets/useWalletInterface";
+import { WalletInterface } from "@/services/wallets/walletInterface";
+import { GroupBase } from "react-select";
+import { useReadIdentityRegistryInvestorCountry } from "@/hooks/useReadIdentityRegistryInvestorCountry";
+
+export const investorCountriesItems = [
+  {
+    value: 840,
+    label: "US",
+  },
+  {
+    value: 0,
+    label: "Non US",
+  },
+];
+
+type Props = {
+  setUpdateTxResult: (res?: string) => void;
+  setUpdateTxError: (err?: string) => void;
+  isOpen: boolean;
+  onClose: () => void;
+  registry?: EvmAddress;
+  selectedIdentity?: {
+    walletAddr: EvmAddress;
+    identityAddr: EvmAddress;
+  };
+};
+
+export const ManageIdentities = ({
```

Comment:
I'd suggest following the codebase with exporting default function instead of const just for consistency